app/Models/GlobalIndicator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+class GlobalIndicator extends Model
+{
+    protected $table = 'global_indicators';
+
+    protected $guarded = ['id'];
+
+    public function theme(): BelongsTo
+    {
+        return $this->belongsTo(Theme::class);
+    }
+
+    public function localIndicators(): BelongsToMany
+    {
+        return $this->belongsToMany(LocalIndicator::class)->withTimestamps();
+    }
```

Comment:
When I saw the setup on the page, I realised I was wrong when writing the issue. A local indicator will be linked to at most 1 global indicator, so this would be a hasMany instead of a belongsToMany.